packages/orbit-components/src/BadgeList/BadgeListItem/index.tsx

```diff
@@ -28,23 +36,17 @@ export const ItemWrapper = ({ children, dataTest }) => (
   </li>
 );
 
-export const VerticalBadge = ({
-  children,
-  type,
-}: {
-  children: React.ReactNode;
-  type: Props["type"];
-}) => {
+export const VerticalBadge = ({ icon, type }: { icon: React.ReactNode; type: Props["type"] }) => {
```

Comment:
I assume this is not a breaking change because it's an internal component, not used anywhere else.

Reply:
Exactly!

Reply:
Cool, resolving.